pkg/plugins/common/kustomize/v1/scaffolds/init.go

```diff
@@ -62,23 +62,44 @@ func (s *initScaffolder) Scaffold() error {
 		machinery.WithConfig(s.config),
 	)
 
-	return scaffold.Execute(
-		&rbac.Kustomization{},
-		&rbac.AuthProxyRole{},
-		&rbac.AuthProxyRoleBinding{},
-		&rbac.AuthProxyService{},
-		&rbac.AuthProxyClientRole{},
-		&rbac.RoleBinding{},
-		&rbac.LeaderElectionRole{},
-		&rbac.LeaderElectionRoleBinding{},
-		&rbac.ServiceAccount{},
-		&manager.Kustomization{},
-		&manager.Config{Image: imageName},
-		&manager.ControllerManagerConfig{},
-		&kdefault.Kustomization{},
-		&kdefault.ManagerAuthProxyPatch{},
-		&kdefault.ManagerConfigPatch{},
-		&prometheus.Kustomization{},
-		&prometheus.Monitor{},
-	)
+	if s.config.IsComponentConfig() {
```

Comment:
@laxmikantbpandhare, 

Could we just add here https://book.kubebuilder.io/component-config-tutorial/api-changes.html the step to add this manifest manually when the project was not created with --component-config?

Reply:
working on it.

Reply:
Updated the docs.